scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -1436,6 +1448,13 @@ async def _wait_for_assistant_reply(
                 text = await assistant.inner_text(timeout=1000)
             except Exception:
                 text = ""
+            try:
+                last_final_reply_state = await assistant.get_attribute(
+                    "data-final-reply",
+                    timeout=1000,
+                )
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)